templates/environment/cf.yml

```diff
@@ -35,6 +35,9 @@ Parameters:
     Default: true
     AllowedValues: [ true, false ]
 
+  EnvironmentManagerPrincipalArn:
```

Comment:
Maybe just for consistency with how we're talking about this, we can call this "ToolsAccountPrincipal"